dask_expr/_expr.py

```diff
@@ -1285,7 +1265,35 @@ class Elemwise(Blockwise):
     optimizations, like `len` will care about which operations preserve length
     """
 
-    pass
+    _filter_passthrough = True
+
+    def _simplify_up(self, parent):
+        if self._filter_passthrough and isinstance(parent, Filter):
+            return type(self)(
+                self.frame[parent.operand("predicate")], *self.operands[1:]
+            )
+        return super()._simplify_up(parent)
+
+
+class RenameFrame(Elemwise):
+    _parameters = ["frame", "columns"]
+    _keyword_only = ["columns"]
+    operation = M.rename
+
+    def _simplify_up(self, parent):
+        if isinstance(parent, Projection) and isinstance(
+            self.operand("columns"), Mapping
+        ):
+            reverse_mapping = {val: key for key, val in self.operand("columns").items()}
+            if is_series_like(parent._meta):
+                # Fill this out when Series.rename is implemented
+                return
+            else:
+                columns = [
+                    reverse_mapping[col] if col in reverse_mapping else col
+                    for col in parent.columns
+                ]
+            return type(self)(self.frame[columns], *self.operands[1:])
```

Comment:
Note that you can also do something like `return self.substitute_parameters({"frame": self.frame[columns]})` in cases like this (which is maybe more readable?).